mongodbatlas/table_mongodbatlas_x509_auth_database_user.go

```diff
@@ -0,0 +1,112 @@
+package mongodbatlas
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+func tableAtlasX509Auth(_ context.Context) *plugin.Table {
```

Comment:
```suggestion
func tableMongoDBAtlasX509AuthenticationDatabaseUser(_ context.Context) *plugin.Table {
```

It's a bit long, but easier to stay consistent with having the function names match the table names